src/lib/faction/faction.data.js

```diff
@@ -42,7 +42,8 @@ const FACTION_DATA = [
         promissoryNotes: ["strike_wing_ambuscade"],
         icon: "global/factions/argent_icon.png",
         source: "pok",
-        startingTech: ["sarween_tools", "neural_motivator", "plasma_scoring"],
```

Comment:
I removed these since @darrellanderson said, the optional are not handed out, but can reintroduce them for this faction (where it is more a "put one tech back"